### PR TITLE
[doc] add section for search experience in tracing doc

### PR DESCRIPTION
**Pull request by zhengfeiwang**
# Description

This pull request adds brief introduction for trace UI search experience.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: docs/how-to-guides/tracing/index.md [hunk 1]
```diff
@@ -117,4 +117,14 @@ pf run create --flow . --data "./data/bert-paper-qna.jsonl" --column-mapping cha
...
+
+### Search
+
+Trace UI supports simple Python expression for search experience, which is demonstrated in below GIF:
+
+![advanced_search](../../media/trace/advanced-search.gif)
+
+Currently it supports bool operator `and` and `or`, compare operator `==`, `!=`, `>`, `>=`, `<`, `<=`; and the fields that are searchable: `name`, `kind`, `status`, `start_time`, `cumulative_token_count.total`, `cumulative_token_count.prompt` and `cumulative_token_count.completion`. You can find the hints by clicking the button right to the search edit box.
+
+![search_hint](../../media/trace/trace-ui-search-hint.png)
```
Comment (wangchao1230) [inline docs/how-to-guides/tracing/index.md:130]: where do we plan to add delete doc?

File: docs/media/trace/trace-ui-search-hint.png
Comment (wangchao1230) [inline docs/media/trace/trace-ui-search-hint.png:1]: the arrows seems a bit sloppy, is it by design?

**Review by wangchao1230 — DISMISSED** [review]

File: docs/how-to-guides/tracing/index.md [hunk 2]
```diff
@@ -117,4 +117,14 @@ pf run create --flow . --data "./data/bert-paper-qna.jsonl" --column-mapping cha
 ```
 Then you will get a run related trace URL, e.g. http://localhost:<port>/v1.0/ui/traces?run=chat_with_pdf_20240226_181222_219335
 
-![batch_run_record](../../media/trace/batch_run_record.png)
\ No newline at end of file
+![batch_run_record](../../media/trace/batch_run_record.png)
+
+### Search
```
Comment (wangchao1230) [inline docs/how-to-guides/tracing/index.md:122]: need to add change log, so user know this is added in 1.10

File: docs/media/trace/trace-ui-search-hint.png
Comment (zhengfeiwang) [reply docs/media/trace/trace-ui-search-hint.png:1]: maybe i can remove that... which is manually added by my hand

File: docs/how-to-guides/tracing/index.md (hunk 1 above)
Comment (zhengfeiwang) [reply docs/how-to-guides/tracing/index.md:130]: good catch, let me add in this pr

Comment (zhengfeiwang) [reply docs/how-to-guides/tracing/index.md:130]: added, may take a look

File: docs/media/trace/trace-ui-search-hint.png
Comment (zhengfeiwang) [reply docs/media/trace/trace-ui-search-hint.png:1]: updated the screenshot

File: docs/how-to-guides/tracing/index.md (hunk 2 above)
Comment (zhengfeiwang) [reply docs/how-to-guides/tracing/index.md:122]: i will prepare a separate pr for changelog update for this

**Review by wangchao1230 — APPROVED** [review]

**Review by crazygao — APPROVED** [review]

**Pull request merged**